src/systems/ec/implementations/vanilla/packages/ec-component-form-checkbox/ec-component-form-checkbox.scss

```diff
@@ -0,0 +1,91 @@
+/**
+ * Checkbox component
+ * @define checkbox
+ */
+
+// Import base
+@import '@ecl/ec-base/ec-base';
+
+@mixin ecl-checkbox(
+  $thickness-base: 1px,
+  $thickness-max: 3px,
+  $border-color: $ecl-color-grey-25,
+  $border-color-focus: $ecl-color-yellow-100,
+  $border-color-checked: $ecl-color-blue-100,
+  $fake-checkbox-size: 0.875rem
+) {
+  .ecl-checkbox {
+    align-items: center;
+    display: flex;
+    position: relative;
+  }
+
+  .ecl-checkbox__input {
+    cursor: pointer;
+    margin-left: 3px;
+    opacity: 0;
+    padding: 0;
+    position: absolute;
+    z-index: 1;
+  }
+
+  .ecl-checkbox__label {
+    color: $ecl-color-grey-100;
+    font-size: $ecl-font-m;
+    font-weight: normal;
+    padding: 2px 0 0 $ecl-spacing-xl;
+  }
+
+  .ecl-checkbox__icon {
+    border: $thickness-base solid $border-color;
+    cursor: pointer;
+    display: inline-flex;
+    left: 0;
```

Comment:
can you explain why `left: 0;`? I don't get it 👼

Reply:
or ignore it, since it might go away with the refactoring of the markup

Reply:
It's just unneeded, so in a way or another it will go away :)